src/main/plugins/org.dita.base/build_init.xml

```diff
@@ -22,6 +22,7 @@ See the accompanying LICENSE file for applicable license.
   <taskdef name="pipeline" classname="org.dita.dost.ant.ExtensibleAntInvoker"/>
   <taskdef name="dita-ot-echo" classname="org.dita.dost.ant.DITAOTEchoTask"/>
   <taskdef name="dita-ot-fail" classname="org.dita.dost.ant.DITAOTFailTask"/>
+  <!-- DEPRECATED: dita-ot-copy replaced with org.dita.dost.module.CopyFlagModule -->
```

Comment:
For the deprecated notes, add the version of OT when it was deprecated.

Reply:
Meant to ask about that -- do I list a hotfix 3.3.3 or the next release, 3.4?